src/Microsoft.AspNetCore.Authentication.Core/AuthenticationService.cs

```diff
@@ -89,14 +90,17 @@ public AuthenticationService(IAuthenticationSchemeProvider schemes, IAuthenticat
                 scheme = defaultChallengeScheme?.Name;
                 if (scheme == null)
                 {
-                    throw new InvalidOperationException($"No authenticationScheme was specified, and there was no DefaultChallengeScheme found.");
+                    context.Response.StatusCode = StatusCodes.Status401Unauthorized;
```

Comment:
So illegal that many clients crash/throw when they don't get the mandatory `WWW-Authenticate` header :smile: